java/src/shared/model/game/TurnTracker.java

```diff
@@ -61,11 +68,41 @@ public int getCurrentTurn() {
     }
 
     /**
+     * Goes to the next players turn during the setup phase.
```

Comment:
could we also get a function checking if it the players turn during the setup round? so passes in a playerID, and checks if it's that's player's turn